### ISSUE TITLE
Imrovements to properties documentation

### ISSUE THREAD

**Issue by slavizh** [microsoftgraph/msgraph-bicep-types#112]
**Is your feature request related to a problem? Please describe.**
There are things that needs to be improved into documentation of properties around the default values if the property is not specified. 
Example 1:
![image](https://github.com/microsoftgraph/msgraph-bicep-types/assets/11890856/800b7b9c-2809-4234-b76f-a4dab29d66aa)

I cannot understand what is the default value of this property. Is it true or it is false. Can also the default to be null? Or may be the default depends on how other properties are configured. It will help a lot at the end of this description we have something simple as Default: true.

Example 2:
![image](https://github.com/microsoftgraph/msgraph-bicep-types/assets/11890856/f2016de1-dd7e-4e8c-b065-84e485c7d377)

Also unclear if default value is something generic as en-US or null.

Such examples we can find on other properties.
 
**Describe the solution you'd like**

**Additional context**

**Comment by dkershaw10** [comment]
The documentation here is autogenerated based on the existing Microsoft Graph REST API documentation (as the source).  We'll probably need to get that fixed at source.  There are unfortunately many examples of this in the docs, and our API schema is somewhat incomplete in many areas, including defining default property values.

I believe that `isAssignableToRole` can be null, and that is likely the default and that null is interpreted as false.  This is a new-ish property, so will have this sort of behaviour so as not to introduce a breaking change for existing apps.

Bottom line is this is likely to take a while to fix, although will look into some other options too.
